system_helpers.py

```diff
@@ -167,8 +167,8 @@ def determine_league_install_location(override_path: str | None = None) -> str:
             )
             [league_path, _] = winreg.QueryValueEx(access_key, "InstallLocation")
         except Exception as err:
-            logger.error(f"Could not dynamically determine League install location : {str(err)}")
-            logger.error(sys.exc_info())
+            logger.exception(f"Could not dynamically determine League install location : {str(err)}")
+            logger.exception(sys.exc_info())
```

Comment:
I believe if we're going to use the `exception()` convenience method, we should remove the stack trace log since that's already being set:
https://loguru.readthedocs.io/en/stable/_modules/loguru/_logger.html#Logger.exception
https://github.com/Delgan/loguru/issues/235#issuecomment-608014996

```suggestion
```